dev-tools/mage/kubernetes/kuberemote.go

```diff
@@ -164,14 +164,17 @@ func (r *KubeRemote) Run(env map[string]string, stdout io.Writer, stderr io.Writ
 
 // deleteSSHKey deletes SSH key from the cluster.
 func (r *KubeRemote) deleteSSHKey() {
-	_ = r.cs.CoreV1().Secrets(r.namespace).Delete(r.secretName, &metav1.DeleteOptions{})
+	_ = r.cs.CoreV1().Secrets(r.namespace).Delete(context.Background(), r.secretName, metav1.DeleteOptions{})
```

Comment:
Please use `context.TODO()` in the contexts added in this PR. Is the recommendation when a required context is not available yet in the caller. `context.Background()` is intended for initialization of top-level contexts.

Reply:
Btw, all mage targets can receive a context https://magefile.org/targets/

Reply:
👍